### ISSUE TITLE
File separator issues prevent build on Windows

### ISSUE THREAD

**Issue by benedeki** [AbsaOSS/spline#345]
While in most cases Windows nowadays accept `/` as the file separator, sometimes `\` is still needed.
On few plays of the project files the hard-coded `/` prevents build on Windows system. In those cases using `${file.separator}` fixes the issue.

**Comment by wajda** [comment]
Can you please give us more details about the issue? What hardcoded slashed and were prevent build?

**Comment by benedeki** [comment]
Most of build tools actually works OK. But some plugins require OS specific file separator.
E.g.
`rest-gateway/pom.xml` line 144
```
<exec executable="cmd" osfamily="windows" failonerror="true">
  <arg line="/c node_modules${file.separator}.bin${file.separator}redoc-cli.cmd bundle -o ${swagger.output.path}/consumer.html ${swagger.output.path}/consumerSwagger.json"/>
</exec>
```

**Comment by leborgneromain** [comment]
@benedeki I found this bug already and it will be fixed in the next PR.

**Comment by wajda** [comment]
Since Romain got it fixed in his PR already (4ff6db8a7ed82fc5fab1d17eb823378b79e429e5, specifically [here](https://github.com/AbsaOSS/spline/pull/347/files#diff-387b4088e123ef74679c967e667c6c07R144) ) I'd suggest to close it.
